datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/dashboard.py

```diff
@@ -0,0 +1,98 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import json
+import os
+import re
+
+import click
+import requests
+
+from ....utils import ensure_dir_exists, path_join, write_file
+from ...constants import get_root
+from ...utils import get_valid_integrations, load_manifest, write_manifest
+from ..console import CONTEXT_SETTINGS, abort
+
+BOARD_ID_PATTERN = r'datadoghq\.com/[^/]+/([^/]+)/.+'
+SCREEN_API = 'https://api.datadoghq.com/api/v1/screen/'
+
+
+@click.group(context_settings=CONTEXT_SETTINGS, short_help='Dashboard utilities')
+def dash():
+    pass
+
+
+@dash.command(context_settings=CONTEXT_SETTINGS, short_help='Export a Dashboard as JSON')
+@click.argument('url')
+@click.argument('integration', required=False)
+@click.pass_context
+def export(ctx, url, integration):
+    if integration and integration not in get_valid_integrations():
+        abort('Unknown integration `{}`'.format(integration))
+
+    org = ctx.obj['org']
+    if not org:
+        abort('No `org` has been set')
+
+    if org not in ctx.obj['orgs']:
+        abort('Selected org {} is not in `orgs`'.format(org))
+
+    org = ctx.obj['orgs'][org]
+
+    api_key = org.get('api_key')
+    if not api_key:
+        abort('No `api_key` has been set')
+
+    app_key = org.get('app_key')
+    if not app_key:
+        abort('No `app_key` has been set')
+
+    match = re.search(BOARD_ID_PATTERN, url)
```

Comment:
I think it would be easier to ask for the `board_id` as input directly instead of the url.

Reply:
Nah, telling people to just copy/paste the url is way easier